src/pybind/mgr/cephadm/module.py

```diff
@@ -1400,6 +1400,17 @@ def _add_host(self, spec):
             raise OrchestratorError('New host %s (%s) failed check(s): %s' % (
                 spec.hostname, spec.addr, errors))
 
+        # prime crush map?
```

Comment:
should this be done everytime we save the host, not just once?

Reply:
i think just once... i don't want to get into a situation where a user is trying to do ``ceph osd crush move ...`` and has to fight with cephadm.  (OTOH this means an export + import might move the item, but i think that's less bad.)

Reply:
actually, nm, ``add-bucket`` succeeds if the item exists in the wrong position, so i think it works fine as-is